util/problem_detail.py

```diff
@@ -53,6 +52,14 @@ def detailed(self, detail, status_code=None, title=None, instance=None,
 
         The detailed error message will be shown to patrons.
         """
+
+        # Title and detail must be LazyStrings from Flask-Babel that are
+        # localized when they are first used as strings.
+        if title and not isinstance(title, LazyString):
+            raise Exception("\"%s\" has not been internationalized" % title)
```

Comment:
Raising an exception here will create a 500 error and make the patron think something is seriously wrong. I'd prefer to just log a warning. We can set up an alert for "has not been internationalized".